bindings/python/fdb/directory_impl.py

```diff
@@ -613,17 +612,17 @@ def _is_prefix_empty(self, tr, prefix):
 
 def _to_unicode_path(path):
```

Comment:
Python3 has a `pathlib.Path` which is lovely. It would be wonderful if we can use it.

Reply:
Sounds good ^-^ One problem could be bytes though, not sure if it is used that way but `_to_unicode_path()` handles `bytes` as well, along with `tuple of bytes` -- it converts `a tuple of bytes` to `a tuple of str`. Perhaps also used for paths received from FDB? 

The other thing I am a bit worried that this might be a bit bigger change since it is part of the API: https://apple.github.io/foundationdb/api-python.html#fdb.DirectoryLayer.create_or_open

Reply:
Maybe you can leave an issue, assign it to me (so I can take some more look later, currently on some other nastiy issues) and ignore this?